decentraland/unity-renderer
# fix: scroll sensibility across uis

## What does this PR change?

Properly configured the scroll through all the UIs to get a better sensibility across different platforms.
Added a prefab validation test for it.

## How to test the changes?

1. WebGL: https://play.decentraland.org/?explorer-branch=fix/scroll-sens
2. Desktop: https://renderer-artifacts.decentraland.org/desktop/index.html?dcl://DESKTOP-BRANCH=fix/scroll-sens
3. Go through all the UIs and use the scroll. It should have a proper sensibility on each platform: webgl/desktop/mac/windows/linux
4. Example UIs: explore, friends, chat, map, settings, terms & conditions (registration process), voice chat list, experiences viewer, passport, dropdowns, carousels.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
